lib/node_modules/@stdlib/math/base/special/boxcox/docs/img/equation_box_cox_transformation_one_parameter.svg

```diff
@@ -0,0 +1,84 @@
+<svg xmlns:xlink="http://www.w3.org/1999/xlink" width="27.261ex" height="8.843ex" style="vertical-align: -3.838ex;" viewBox="0 -2154.8 11737.4 3807.2" role="img" focusable="false" xmlns="http://www.w3.org/2000/svg" aria-labelledby="MathJax-SVG-1-Title">
```

Comment:
@kgryte I thought had moved the image that I made from LaTex using codecogs.com, as I had a placeholder image from wikimedia commons. However, I regrettably must have not overwrite that placeholder image. Should I remove the image since there is an automated process?

Reply:
@justin1dennison Yeah, you can go ahead and just remove the image. We can generate it on merge. Thanks!